canonical-websites/build.snapcraft.io
# 866 create search field component

## Done

- Created the new search field component for the repositories list
- Included styles to align content in the add repos header

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Go to the add repos page and the new search component shall show in the header of the over all module.


## Issue / Card

#866
